tsup.config.ts

```diff
@@ -0,0 +1,40 @@
+import { defineConfig } from 'tsup';
+
+export default defineConfig({
+  // The entry to bundle.
+  entry: [
+    'src/**/*.ts',
+    '!src/**/__fixtures__/**/*',
```

Comment:
It seems that we would have to keep this in sync with TypeScript and Jest configuration. I don't have any suggestions here but it would be nice if we didn't have to do this because I can see this getting forgotten. I guess the good thing is that we tend to check the files inside of a published package before it goes out.

Reply:
Yeah, not much we can do about that I think. I added the most common files and folders, so in most cases we won't have to modify it.